cdap-docs/_common/common-build.sh

```diff
@@ -258,32 +259,35 @@ function test_an_include() {
   local md5_hash=${1}
   local target=${2}
   local new_md5_hash
+  local m
+  local m_display  
   
-  local file_name=`basename ${target}`
-  
-  if [[ "${OSTYPE}" == "darwin"* ]]; then
-    new_md5_hash=`md5 -q ${target}`
-  else
-    new_md5_hash=`md5sum ${target} | awk '{print $1}'`
-  fi
+  if [[ "x${target}" == "x" ]]; then
+    m="No target is set for test_an_include"
+  else  
+    local file_name=`basename ${target}`
```

Comment:
I would have gone the other way, but the main thing was being consistent. Back-ticks are fine.